### PR TITLE
updated docs and migrated screenshots

### PR DESCRIPTION
**Pull request by JKashyap96**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/google/compute-engine.md [hunk 1]
```diff
@@ -314,6 +314,8 @@ See the details of projects, zones, instances, and message severity in your Goog
 
 See the details of message severities - emergency, alert, critical, error, and warning.
 
+<img src={useBaseUrl('img/integrations/google/google-compute-engine-severity.png')} alt="Google Compute Engine dashboards" />
```
Comment (JV0812) [inline docs/integrations/google/compute-engine.md:317]: Image is not added. It is broken when previewed, kindly check.

File: docs/integrations/amazon-aws/ses.md [hunk 2]
```diff
@@ -289,7 +289,7 @@ See the overview of SES CloudTrail events including failed, and successful event
 
 See the details of various SES CloudTrail events including the identity, get send, domain, receipt, and email address events.
 
-<img src={useBaseUrl('img/integrations/amazon-aws/SES-Notification.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESCloudTrailEventsbyEventName.png')} alt="AWS SES Cloud Trail Events by Event Name" />
```
Comment (JV0812) [inline docs/integrations/amazon-aws/ses.md:292]: It is broken when previewed, kindly check.

File: docs/integrations/amazon-aws/ses.md [hunk 3]
```diff
@@ -258,7 +258,7 @@ This section provides examples of the Amazon SES App dashboards, along with desc
 
 See the overview of SES CloudTrail events including failed, and successful events, error codes, users, and event locations.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ses.md:259]:
```suggestion
See the overview of SES CloudTrail events including failed, successful events, error codes, users, and event locations.
```

File: docs/integrations/amazon-aws/ses.md [hunk 4]
```diff
@@ -258,7 +258,7 @@ This section provides examples of the Amazon SES App dashboards, along with desc
...
 
-<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESCloudTrailEventsbyEventName.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESCloudTrailEventsOverview.png')} alt="AWS SES CloudTrail Events Overview" />
```
Comment (JV0812) [inline docs/integrations/amazon-aws/ses.md:261]: It is broken when previewed, kindly check.

File: docs/integrations/amazon-aws/ses.md [hunk 5]
```diff
@@ -383,7 +383,7 @@ See the details of bounced notifications by email addresses, domains, bounce typ
 
 See information about complaints (a complaint occurs when a recipient reports that they don't want to receive an email), including the top email addresses, email domains, and UserAgents associated with complaints; and the sending AccountId, AWS region, SourceIP, and Identity associated with complaints.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ses.md:384]:
```suggestion
See information about complaints (a complaint occurs when a recipient reports that he doesn't want to receive an email), including the top email addresses, email domains, and user agents associated with complaints and the sender's AccountId, AWS region, SourceIP, and Identity associated with complaints.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]

File: docs/integrations/amazon-aws/ses.md (hunk 5 above)
Comment (JainM6) [reply docs/integrations/amazon-aws/ses.md:384]: Did you mean to write "sender's" instead of sending?

File: docs/integrations/amazon-aws/ses.md [hunk 6]
```diff
@@ -312,11 +312,11 @@ See the details of various SES CloudTrail events including the identity, get sen
...
 
-#### Notification Overview
+### Notification Overview
 
 See the overview of SES notifications including the source IP locations, notification types, mail source, and notification type.
 
-<img src={useBaseUrl('img/integrations/amazon-aws/Overview.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/SES-Notification.png')} alt="AWS SES Notification" />
 
 **Mail Source IP Locations**. See the count and location of the mail source IP addresses in the last 24 hours on a world map.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ses.md:321]:
```suggestion
**Mail Source IP locations**. See the count and location of the mail source IP addresses in the last 24 hours on a world map.
```

File: docs/integrations/amazon-aws/ses.md [hunk 7]
```diff
@@ -352,7 +352,7 @@ See the details of delivered notifications including the email destinations, dom
 
 See the details of bounced notifications by email addresses, domains, bounce types, and bounce subtypes.
 
-<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESBouncedNotifications.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESBouncedNotifications.png')} alt="AWS SES Bounced Notifications" />
 
 **Top Bounced email Addresses.** See the top 10 bounced email addresses by count in the last 24 hours.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ses.md:357]:
```suggestion
**Top Bounced email addresses.** See the top 10 bounced email addresses by count in the last 24 hours.
```

File: docs/integrations/amazon-aws/ses.md [hunk 8]
```diff
@@ -383,7 +383,7 @@ See the details of bounced notifications by email addresses, domains, bounce typ
 
 See information about complaints (a complaint occurs when a recipient reports that they don't want to receive an email), including the top email addresses, email domains, and UserAgents associated with complaints; and the sending AccountId, AWS region, SourceIP, and Identity associated with complaints.
 
-<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESComplaintNotifications.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESComplaintNotifications.png')} alt="AWS SES Complaint Notifications" />
 
 **Top Complaint email Addresses.** See the top 10 email addresses sending emails generating complaint notifications by count in the last 24 hours.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/ses.md:388]:
```suggestion
**Top Complaint email addresses.** See the top 10 email addresses sending emails generating complaint notifications by count in the last 24 hours.
```

File: docs/integrations/amazon-aws/ses.md (hunk 4 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/ses.md:261]: Adding the image.

File: docs/integrations/amazon-aws/ses.md (hunk 2 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/ses.md:292]: https://github.com/SumoLogic/sumologic-documentation/pull/2593/files#diff-7c72755f83e5f7bd574a60da82c3abdb4b72b534bd3763280b18eea09aadb2d2

Already added in the PR.

**Review by JainM6 — CHANGES_REQUESTED** [review]
Minor changes

File: docs/integrations/amazon-aws/ses.md (hunk 6 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/ses.md:321]: Screenshot 2023-06-09 at 4 21 31 PM
Capital 'L' is expected here as it is a defined panel name.

File: docs/integrations/google/compute-engine.md (hunk 1 above)
Comment (JKashyap96) [reply docs/integrations/google/compute-engine.md:317]: adding the image

File: docs/integrations/amazon-aws/ses.md (hunk 5 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/ses.md:384]: Yes, good catch, updating this.

File: docs/integrations/amazon-aws/ses.md (hunk 3 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/ses.md:259]: removing the comma, to say 'failed and successful events'

File: docs/integrations/amazon-aws/ses.md (hunk 7 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/ses.md:357]: Screenshot 2023-06-09 at 4 28 39 PM
It is expected here.

File: docs/integrations/amazon-aws/ses.md (hunk 8 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/ses.md:388]: expected again.

**Comment by JKashyap96** [comment]
@JainM6 @JV0812 Please review again!

File: docs/integrations/amazon-aws/ses.md (hunk 2 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/ses.md:292]: Please check once

**Review by JainM6 — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]

**Pull request merged**